.github/workflows/update-snapshot.yml

```diff
@@ -94,7 +94,7 @@ jobs:
         with:
           args: |
             PR to update the website to snapshot ${{env.SNAPSHOT_ID}} (with updated map colors and share images) is available.
-            PR: https://github.com/covid-projections/covid-projections/pull/${{env.PULL_REQUEST_NUMBER}}
-            Preview: https://covid-projections-git-bump-snapshot-${{env.RUN_ID}}-covidactnow.now.sh/
-            Compare: https://covid-projections-git-bump-snapshot-${{env.RUN_ID}}-covidactnow.now.sh/internal/compare/
+            PR: https://github.com/act-now-coalition/covid-act-now-website/pull/${{env.PULL_REQUEST_NUMBER}}
+            Preview: https://covid-act-now-website-git-bump-snapshot-${{env.RUN_ID}}-covidactnow.now.sh/
+            Compare: https://covid-act-now-website-git-bump-snapshot-${{env.RUN_ID}}-covidactnow.now.sh/internal/compare/
```

Comment:
Same here.

Reply:
Same answer.  Seems like it should still be "covid-projections" for now.